### ISSUE TITLE
reproductiveCondition

### ISSUE THREAD

**Issue by yohanaditzel** [sibbr/DarwinCoreBrasil#30]
https://dwc.tdwg.org/terms/#dwc:reproductiveCondition

Vocabulário controlado recomendando.
Definição: a condição reprodutiva do(s) indivíduo(s) biológico(s) representado(s) na ocorrência.

**Comment by edalcin** [comment]
Creio que o exemplo correto seria "Frutífero" e não "Frutificação"

**Comment by angelitacoelho** [comment]
Geralmente, em estudos fenológicos são usados os termos "Floração" e "Frutificação".

**Comment by ClaraBaringoFonseca** [comment]
Pessoal, bom dia, obrigada pelos comentários. O reproductiveCondition faz relação ao momento da coleta correto? Nesse caso seria se a planta no dia da coleta está em fase de Floração? ou em fase de frutificação? Além disso existem outros termos que deveríamos colocar na parte de plantas? Vou alterar aqui. Obrigada!

**Comment by angelitacoelho** [comment]
Sim, qual a fase do indivíduo no momento da coleta. Como já tem a opção "Não reprodutiva" acho que não é necessária a inclusão de outros termos.

**Comment by ClaraBaringoFonseca** [comment]
Obrigada Angelita! Qualquer recomendação e sugestão de termos é sempre bem-vinda.

**Comment by ClaraBaringoFonseca** [comment]
Pessoal, 
Conversamos com o curador da coleção de mamíferos da UFLA e tinha o seguinte vocabulário controlado:
**Grávida**
**Inativa** seria uma fêmea adulta que não está grávida nem lactante (ou pós-lactante). Nesse caso eu colocaria **Fêmea inativa**
**Lactante** seria uma fêmea que tem mama desenvolvida com leite, mas a fêmea não está grávida, está supostamente amamentando; 
**Pós-lactante**. seria fêmea não grávida, com mamas desenvolvidas, mas estas estão sem leite, "secas".
**Machos escrotados** seriam com o saco escrotal (e testículos) bem evidentes, desenvolvidos.
Alguma sugestão?
Obrigada,

**Comment by ClaraBaringoFonseca** [comment]
Para estudos com morcegos as categorias dividem entre fêmea e macho
Para Fêmeas:
GL = Fêmea grávida simultaneamente lactante; 
GV = Fêmea grávida; 
IN = Fêmea inativa; 
LC = Fêmea lactante; 
NA = Não foi visto
